wordpress/wp-content/mu-plugins/cds-base/classes/Modules/Users/Users.php

```diff
@@ -184,6 +184,25 @@ public function sendReset($uId, $email)
         wp_mail($email, $subject, $message);
     }
 
+    public function sendAddToCollection($uId, $email)
+    {
+        $userInfo = get_userdata($uId);
+
+        $blogName = get_bloginfo('name');
+
+        // phpcs:disable
+        $subject = __("Invitation to collaborate on GC Articles", "cds-snc"). " — ". $blogName;
+        
+        $blogInfo->name;
+        
+        $message = __('Someone has invited this email to collaborate on a GC Articles collection site.', "cds-snc") . "\r\n\r\n";
```

Comment:
It would be good to have a link to the site as well. 

I created a new collection to test this, so I got both of the emails one after the other.

| creating a new site | adding a pre-existing user |
|---------------------|----------------------------|
|     Lots of info in here: new site url, your username, url for login page, two bullet points (🤔 )                |      Almost no info in here. Would be good to have  the same 2 links as the other one: link to site, link to login page.  We can skip the username since it's the same as the email address.                 |
|    <img width="599" alt="Screen Shot 2021-11-25 at 11 35 51" src="https://user-images.githubusercontent.com/2454380/143477920-9c3477e1-10c8-41ab-b6c2-75929d928a47.png">           |  <img width="609" alt="Screen Shot 2021-11-25 at 11 39 08" src="https://user-images.githubusercontent.com/2454380/143477923-ab156125-851e-47f5-bdc4-667bd1b3bc8f.png"> |

The one thing not pictured is that the site name is in the subject line for both, which is good. But getting the URLs would still be part of making this actionable.

Reply:
Added a url + the site name to the messaging.